scripts/qa_crowbarsetup.sh

```diff
@@ -1063,6 +1063,58 @@ EOF
         fi
     fi
 
+    if [[ $want_ironic = 1 ]] ; then
+        local conmap_complete=`
+            python - <<EOPYTHON
+import json
+f=open('$netfile')
+j=json.load(f)
+conduit_map=j['attributes']['network']['conduit_map']
+for item in conduit_map:
+    if item['pattern'] == 'single/.*/.*':
+        item['conduit_list']['intf3'] = {'if_list': ['?1g2']}
+    else:
+        item['conduit_list']['intf3'] = item['conduit_list']['intf1']
+print json.dumps(conduit_map, indent=4)
+EOPYTHON
+        `
+        /opt/dell/bin/json-edit -a attributes.network.conduit_map -r -v "$conmap_complete" $netfile
+        local networks_complete=`python - <<EOPYTHON
+import json
+f=open('$netfile')
+j=json.load(f)
+networks=j['attributes']['network']['networks']
+ironic_network={
+    'conduit': 'intf3',
+    'vlan': 100,
+    'use_vlan': False,
+    'add_bridge': False,
+    'add_ovs_bridge': False,
+    'bridge_name': '${cloud}irobr',
```

Comment:
I think this is wrong, it should be br-ironic because https://github.com/crowbar/crowbar-openstack/blob/88efcc0eb97a54b5e18cd711bb44701c86b41655/chef/cookbooks/neutron/recipes/common_agent.rb#L196

Reply:
Yeah, you're right. `${cloud}irobr` is the bridge on host level simulating the physical ironic network. In my scenario files I have `br-ironic` hardcoded: https://github.com/SUSE-Cloud/automation/blob/master/scripts/scenarios/cloud8/cloud8-ironic-basic-ha.yml#L87.

Reply:
Fixed